pkg/apis/imageregistry/v1/types.go

```diff
@@ -153,16 +158,19 @@ type ImageRegistryStatus struct {
 
 	// StorageManaged is a boolean which denotes whether or not
 	// we created the registry storage medium (such as an
-	// S3 bucket)
+	// S3 bucket).
 	StorageManaged bool `json:"storageManaged"`
-
-	// Storage indicates the current applied storage configuration of the registry
+	// Storage indicates the current applied storage configuration of the
+	// registry.
 	Storage ImageRegistryConfigStorage `json:"storage"`
 }
 
 type ImageRegistryConfigProxy struct {
-	HTTP    string `json:"http"`
-	HTTPS   string `json:"https"`
+	// +optional
+	HTTP string `json:"http"`
+	// +optional
+	HTTPS string `json:"https"`
+	// +optional
```

Comment:
If we make `Proxy` a pointer in `spec`, then we can can keep these as "required".

I will also be adding a PR soon to have a generic Proxy with these fields (and appropriate doc for oc explain)